### PR TITLE
logs the ODE info with level debug

### PR DESCRIPTION
**Pull request by yzmyyff**

### CODE REVIEW

**Comment by lucidrains** [comment]
@yzmyyff ah nice, i'm guessing you are doing a lot of training and inferencing?

File: voicebox_pytorch/voicebox_pytorch.py [hunk 1]
```diff
@@ -28,6 +29,7 @@
 
 from vocos import Vocos
 
+_LOGGER = logging.getLogger(__file__)
```
Comment (lucidrains) [inline voicebox_pytorch/voicebox_pytorch.py:32]: yea looks good, want to just remove that underscore?  `LOGGER` is fine, or even `debug_print = logging.getLogger(__file__).debug`

Comment (yzmyyff) [reply voicebox_pytorch/voicebox_pytorch.py:32]: OK

**Comment by yzmyyff** [comment]
Yes, these prints flush out the other logs...

**Comment by lucidrains** [comment]
@yzmyyff ok thanks a bunch!

**Pull request merged**